UI/window-basic-auto-config.cpp

```diff
@@ -566,7 +566,8 @@ void AutoConfigStreamPage::UpdateKeyLink()
 	if (serviceName == "Twitch") {
 		streamKeyLink =
 			"https://www.twitch.tv/broadcast/dashboard/streamkey";
-	} else if (serviceName == "YouTube / YouTube Gaming") {
+	} else if ((serviceName == "YouTube - RTMP") or
```

Comment:
Could you change or to || for consistency with the rest of the code base? And same below.

Reply:
Are the service names in services.json file also updated with spaces around -? Are the changes not in this commit?